docker-api/src/main/java/com/yahoo/vespa/hosted/dockerapi/DockerImpl.java

```diff
@@ -40,10 +39,11 @@
 import java.util.Set;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
 import java.util.logging.Logger;
 import java.util.stream.Stream;
 
-public class DockerImpl implements Docker {
+public class DockerImpl implements ContainerEngine {
```

Comment:
MIght as well simplify this to `Docker` now.

Reply:
DockerEngine?